crates/schema/src/def.rs

```diff
@@ -937,6 +960,18 @@ impl ModuleDefLookup for ConstraintDef {
     }
 }
 
+impl ModuleDefLookup for RowLevelSecurityDef {
+    type Key<'a> = &'a Identifier;
```

Comment:
What is `ModuleDefLookup` used for? Does `RowLevelSecurityDef` need to implement it? If so, could the `Key` just be the sql string itself?